src/auto/unix_socket_address.rs

```diff
@@ -0,0 +1,151 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use SocketAddress;
+use SocketConnectable;
+use UnixSocketAddressType;
+use ffi;
+use glib;
+use glib::StaticType;
+use glib::Value;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct UnixSocketAddress(Object<ffi::GUnixSocketAddress, ffi::GUnixSocketAddressClass>): SocketAddress, SocketConnectable;
+
+    match fn {
+        get_type => || ffi::g_unix_socket_address_get_type(),
+    }
+}
+
+impl UnixSocketAddress {
+    pub fn new(path: &str) -> UnixSocketAddress {
```

Comment:
Is this necessarily a string? AFAIU this is always a `Path`

Reply:
Looks like it.

Reply:
Is it okay if I do a manual implementation for now?

Reply:
Totally.